base/reduce.jl

```diff
@@ -396,7 +396,11 @@ end
 
 
 ## prod
+"""
+    prod(f, iter)
```

Comment:
Thanks, Michael! Both of those seem to work great. My concern now is that the method I'm documenting shows up as `prod(f::Callable, itr)`, though `Callable` is not exported. The equivalent method for `sum` does not have `Callable` in its signature in the online documentation (see [here](http://docs.julialang.org/en/stable/stdlib/collections/)), and the `sum` method does not show up at all when I build the docs locally. Any idea what's going on here?